marthamareal/ridemyway-api-django
# #16397681 Add a pull request template

#### What does this PR do?
- Creates a template for PRs of the project.

#### Description of Task to be completed?
- Add a .github folder and a file inside it that contains the template headings

#### How should this be manually tested?
- Create a pull request from one branch to another, you will automatically see the headings of the PR written for you.

#### Any background context you want to provide?
- The file must be specifically named `pull_request_template`.`md or txt`. You can also use uppercase for the file name but not the extensions.

#### What are the relevant pivotal tracker stories?
- [ #16397681](https://www.pivotaltracker.com/story/show/163976081)
